### PR TITLE
fix(docs): change bash to shell in code language blocks

### PR DESCRIPTION
**Pull request by muescha**
## Description

changed:

- change `bash` to `shell` regarding the [code style guides to prefer `shell`](https://www.gatsbyjs.org/contributing/gatsby-style-guide/#code-formatting-type-tab)

## Related links

- https://www.gatsbyjs.org/contributing/gatsby-style-guide/#code-formatting-type-tab

### CODE REVIEW

File: packages/gatsby-remark-prismjs/README.md [hunk 1]
```diff
@@ -380,14 +380,14 @@ By default, every line gets a prompt appended to the start, this behaviour can b
 to the language.
 
 ````
-```bash{outputLines: 2-10,12}
+```shell{outputLines: 2-10,12}
```
Comment (LekoArts) [inline packages/gatsby-remark-prismjs/README.md:383]: As shown in line 376 this needs to be `bash`

File: packages/gatsby-remark-prismjs/README.md [hunk 2]
```diff
@@ -380,14 +380,14 @@ By default, every line gets a prompt appended to the start, this behaviour can b
...
 ````
 
 The user and host used in the appended prompt is pulled from the `prompt.user` and `prompt.host` values,
 unless explicitly overridden by the `promptUser` and `promptHost` options in the snippet, e.g.:
 
 ````
-```bash{promptUser: alice}{promptHost: dev.localhost}
+```shell{promptUser: alice}{promptHost: dev.localhost}
```
Comment (LekoArts) [inline packages/gatsby-remark-prismjs/README.md:390]: Same ^

**Comment by muescha** [comment]
good catch.

in line 209 it says it is in both:
https://github.com/gatsbyjs/gatsby/blob/2a7cb8c10dff4f847ca8e13db2486ad877c80dfb/packages/gatsby-remark-prismjs/README.md#L209-L210

maybe (in a new PR) would be good to add the `shell` here:

https://github.com/gatsbyjs/gatsby/blob/2f0044d5dcc43b4570cac8945f61edf3dec0f5bc/packages/gatsby-remark-prismjs/src/index.js#L91-L96

```diff
-    [`bash`].includes(languageName) &&
+    [`bash`,`shell`].includes(languageName) &&
```

File: packages/gatsby-remark-prismjs/README.md (hunk 1 above)
Comment (muescha) [reply packages/gatsby-remark-prismjs/README.md:383]: fixed in:

- #23256 `fix(gatsby-remark-prismjs): Update useCommandLine`

File: packages/gatsby-remark-prismjs/README.md (hunk 2 above)
Comment (muescha) [reply packages/gatsby-remark-prismjs/README.md:390]: fixed in:

- #23256 `fix(gatsby-remark-prismjs): Update useCommandLine`

File: packages/gatsby-remark-prismjs/README.md (hunk 1 above)
Comment (muescha) [reply packages/gatsby-remark-prismjs/README.md:383]: updated line 376 in df7ad89

File: docs/blog/2020-01-08-git-workflows/index.md [hunk 3]
```diff
@@ -84,7 +84,7 @@ Our `pt1` PR is approved (🎉), but now we have to reconcile our `pt2` branch.
 
 We check out the `pt2` branch and rebase it onto the updated `pt1` branch:
 
-```bash
+```shell
 $ git checkout feat/headless-cms-pt2
 $ git rebase feat/headless-cms-pt1
```
Comment (Ekwuno) [inline docs/blog/2020-01-08-git-workflows/index.md:89]:
```suggestion
 git checkout feat/headless-cms-pt2
 git rebase feat/headless-cms-pt1
```

File: docs/blog/2020-01-08-git-workflows/index.md [hunk 4]
```diff
@@ -103,7 +103,7 @@ In this alternative flow, we're _leveraging_ Git to show us what work needs to b
 
 Once we've fixed all the conflicts, we can finish up our rebase by running the following:
 
-```bash
+```shell
 # stage all the changes we just made
 $ git add .
```
Comment (Ekwuno) [inline docs/blog/2020-01-08-git-workflows/index.md:108]:
```suggestion
git add .
```

File: docs/blog/2020-01-08-git-workflows/index.md [hunk 5]
```diff
@@ -127,7 +127,7 @@ You'll notice that our `C` commit—the only commit in our `pt2` branch—has be
 
 Because we've rewritten the history, by turning `C` into `E`, we need to force-push to update our PR on GitHub:
 
-```bash
+```shell
 $ git push origin feat/headless-cms-pt2 -f
```
Comment (Ekwuno) [inline docs/blog/2020-01-08-git-workflows/index.md:131]:
```suggestion
 git push origin feat/headless-cms-pt2 -f
```

File: docs/blog/2020-01-08-git-workflows/index.md [hunk 6]
```diff
@@ -226,7 +226,7 @@ The Git history pollution isn't a huge deal, since we'll have the chance to squa
 
 If you do wind up squash-merging a branch, you'll need to manually snip out the duplicate commits. You can do this with an [interactive rebase](https://hackernoon.com/beginners-guide-to-interactive-rebasing-346a3f9c3a6d):
 
-```bash
+```shell
 $ git checkout feat/headless-cms-pt2
 $ git rebase -i feat/headless-cms
```
Comment (Ekwuno) [inline docs/blog/2020-01-08-git-workflows/index.md:231]:
```suggestion
 git checkout feat/headless-cms-pt2
 git rebase -i feat/headless-cms
```

File: docs/blog/2020-01-08-git-workflows/index.md [hunk 7]
```diff
@@ -240,7 +240,7 @@ The work we're doing in this example to migrate to a headless CMS might take a w
 
 To accomplish this, we'll do some more local rebasing:
 
-```bash
+```shell
 # Update our local state
 $ git checkout master
 $ git pull origin master
```
Comment (Ekwuno) [inline docs/blog/2020-01-08-git-workflows/index.md:246]:
```suggestion
  git checkout master
  git pull origin master
```

File: docs/blog/2020-01-08-git-workflows/index.md [hunk 8]
```diff
@@ -240,10 +240,10 @@ The work we're doing in this example to migrate to a headless CMS might take a w
...
-```bash
+```shell
 # Update our local state
-$ git checkout master
-$ git pull origin master
+git checkout master
+git pull origin master
 
 # Rebase our root branch
 $ git checkout feat/headless-cms
```
Comment (MichaelDeBoey) [inline docs/blog/2020-01-08-git-workflows/index.md:249]:
```suggestion
git checkout feat/headless-cms
```

File: docs/blog/2020-01-08-git-workflows/index.md [hunk 9]
```diff
@@ -103,9 +103,9 @@ In this alternative flow, we're _leveraging_ Git to show us what work needs to b
...
 Once we've fixed all the conflicts, we can finish up our rebase by running the following:
 
-```bash
+```shell
 # stage all the changes we just made
-$ git add .
+git add .
 
 # wrap up the rebase
 $ git rebase --continue
```
Comment (MichaelDeBoey) [inline docs/blog/2020-01-08-git-workflows/index.md:111]:
```suggestion
git rebase --continue
```

**Comment by muescha** [comment]
i think all requested changes are solved...

**Comment by muescha** [comment]
merged from master

**Comment by muescha** [comment]
🎉

**Pull request merged**